components/provisioner/internal/provisioning/service_test.go

```diff
@@ -238,20 +238,20 @@ func TestService_DeprovisionRuntime(t *testing.T) {
 	t.Run("Should start Runtime deprovisioning and return operation ID", func(t *testing.T) {
 		//given
 		sessionFactoryMock := &sessionMocks.Factory{}
-		readWriteSession := &sessionMocks.ReadWriteSession{}
+		readSession := &sessionMocks.ReadWriteSession{}
```

Comment:
Shouldn't we use sessionMocks.ReadSession instead?

Reply:
Indeed. I fixed it